### ISSUE TITLE
Options to docker-compose should be an Array, not a String

### ISSUE THREAD

**Issue by rbelnap** [puppetlabs/puppetlabs-docker#664]
## Use Case
Multiple options or options with parameters should be supported

```
  docker_compose { $name:
    ensure        => present,
    compose_files => [
      "${full_project_path}/${deploy_env}/docker-compose.yml",
    ],
    options => "--env-file ${full_project_path}/${deploy_env}/.env",

  }
```
Currently fails, because "--env-file path/to/env" is 2 items.  It is also not possible to specify multiple options for the same reason.  

## Describe the Solution You Would Like
This should be an array, which looks to work as expected by just changing the validation (arguments are inserted by the provider correctly already).  This would change the above to:
```
  docker_compose { $name:
    ensure        => present,
    compose_files => [
      "${full_project_path}/${deploy_env}/docker-compose.yml",
    ],
    options => ["--env-file", "${full_project_path}/${deploy_env}/.env"],
  }
```

**Comment by zeha** [comment]
I'll second this, as we also need to set --env-file, and as a string this doesn't work.

**Comment by zeha** [comment]
For the next person looking, this is fixed in 3.13.x and newer
